editor/components/inserter-with-shortcuts/index.js

```diff
@@ -18,18 +18,40 @@ import { withDispatch, withSelect } from '@wordpress/data';
 import BlockIcon from '../block-icon';
 import './style.scss';
 
+function filterItems( items ) {
+	return reject( items, ( item ) =>
+		item.name === getDefaultBlockName() && isEmpty( item.initialAttributes )
+	);
+}
+
+function orderItems( items ) {
+	/**
+	 * Sort items in the following order:
+	 *
+	 * 1. Blocks that are contextually useful (utility = 3)
+	 * 2. Blocks that have been previously inserted (utility = 2)
+	 * 3. Blocks that are in the common category (utility = 1)
+	 * 4. All other blocks (utility = 0)
+	 *
+	 * @see selectors.getInserterItems()
+	 */
+	return orderBy( items, [ 'utility', 'frecency' ], [ 'desc', 'desc' ] );
```

Comment:
There are multiple components sorting by `utility` and `frecency`, and I'd like to provide the same sort in the block completer (#6067) for a consistent insertion experience. Is there a good place we could share this knowledge rather than duplicating?

Reply:
How do you feel about `getInserterItems` doing the sorting by default? `InserterMenu` would have to disable this though so that we don't double sort.

Reply:
I think that would mask our intention for the inserters to use the same default sort and introduce complexity by requiring a way to disable default sort. What do you think of exporting a default sort function from `editor/components/inserter` and having other inserter modules explicitly import it? Then the code would say what we mean which is "Other insertion-related components use the same default sort as the primary insertion menu".

Reply:
I made `getInserterItems` always sort the items in e9fbe66f5. It turns out that my claim that we'd have to make `InserterMenu` disable this was flat out wrong. Having the selector sort the items makes sense to me because:

- It's basically always what you want to do
- It's more performant to sort the items within `createSelector` where the results are cached